sound/soc/sof/ipc3-topology.c

```diff
@@ -2379,6 +2399,7 @@ const struct ipc_tplg_ops ipc3_tplg_ops = {
 	.widget_free = sof_ipc3_widget_free,
 	.widget_setup = sof_ipc3_widget_setup,
 	.dai_config = sof_ipc3_dai_config,
+	.dai_config_ipc = sof_ipc3_dai_config_ipc,
```

Comment:
What is stopping the ipc4 implementation of `.dai_config` ops to just not send the IPC?
If the `data` parameter is not NULL, it sets the config (and don't send an IPC) if the `data` parameter is NULL then it just return with 0 (and similarly don't send and IPC)?

I don't think it is a good idea to introduce a split when it can be handled just fine with the existing infra.